projects/e2e/workbench-application-platform/src/view.e2e-spec.ts

```diff
@@ -651,6 +652,33 @@ describe('View', () => {
       // expect no 'postMessage' error to be thrown
       await expect(hostAppPO.hasBrowserError('Failed to execute \'postMessage\' on \'DOMWindow\'')).toBeFalsy();
     });
+
+    it('should allow an application to interact with the host application once navigated away from its root page [testcase: c985a55bViewPO-view]', async () => {
```

Comment:
Please also reference the issue in the brackets, e.g. [testcase: c985a55bViewPO-view, issue: #141]